test/metadata_test.rb

```diff
@@ -84,5 +84,42 @@ def setup
       assert_equal "Friendly Name", req_attr.attribute("FriendlyName").value
       assert_equal "Attribute Value", REXML::XPath.first(xml_doc, "//md:AttributeValue").text.strip
     end
+
```

Comment:
I was following the convention of the other tests in this file, but I can refactor this with the cleaner format. Should I refactor the existing tests as well?